Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -14,12 +14,21 @@ public class PlayerMovement : MonoBehaviour
 
     public bool isSneaking;
     public bool isSprinting;
+    public bool isJumping;
```

Comment:
As mentioned in the [programming conventions](https://docs.google.com/document/d/1ngzQ6UUjyCyknk-3t2Rl_pwRQeF-trg98L_ouxy9t1c/edit#heading=h.wkccxurzibci), members should be private wherever possible.

If these do not need to be accessible to other scripts, consider changing them to be `private`.
If they do not need to be accessible to other scripts, but do need to be accessible in the editor, use `[SerializeField] private bool ...`